src/services/wishlist/indext.ts

```diff
@@ -0,0 +1,42 @@
+import staticToken from "assets/data";
+import axios from "axios";
+import { ADD_TO_WISHLIST, REMOVE_FROM_WISHLIST } from "types/cart";
+
+export const addToWishlist = async (product: any, wishlistDispatch: any) => {
+  try {
+    await axios.post(
+      "/api/user/wishlist",
```

Comment:
can it be moved to `const WishListBaseURL = '/api/user/wishlist'`

Reply:
Yes I'll update that